src/main/java/me/jellysquid/mods/sodium/client/render/chunk/compile/ChunkBuilder.java

```diff
@@ -342,8 +342,18 @@ public void run() {
                     continue;
                 }
 
-                // Perform the build task with this worker's local resources and obtain the result
-                ChunkBuildResult result = job.task.performBuild(this.pipeline, this.bufferCache, job);
+                ChunkBuildResult result;
+                try {
```

Comment:
Style: One blank line between the start of a new block and a previous non-comment line

```suggestion

                try {
```